packages/runtime/test/transport/relationships.test.ts

```diff
@@ -338,9 +338,10 @@ describe("RelationshipTermination", () => {
         await syncUntilHasRelationships(services2.transport);
         const incomingRequest = (await services2.eventBus.waitForEvent(IncomingRequestReceivedEvent)).data;
 
-        const canAcceptResult = (await services2.consumption.incomingRequests.canAccept({ requestId: incomingRequest.id, items: [{ accept: true }] })).value;
-        expect(canAcceptResult.isSuccess).toBe(false);
-        expect(canAcceptResult.code).toBe("error.consumption.requests.noMatchingRelationship");
+        const canAcceptResult = await services2.consumption.incomingRequests.canAccept({ requestId: incomingRequest.id, items: [{ accept: true }] });
+        expect(canAcceptResult).toBeAnError(/.*/, "error.consumption.requests.noMatchingRelationship");
+        // expect(canAcceptResult.isSuccess).toBe(false);
+        // expect(canAcceptResult.code).toBe("error.consumption.requests.noMatchingRelationship");
```

Comment:
commented out code?

Reply:
I didn't mean to commit that - whoops.
But if you've seen this: this test doesn't work, the result is successful. I've not yet found a way to make this work - and the requests tests also do it my previous way

Reply:
Ok, i'd say return it as it was and we'll do a refactoring afterwards.